tools/util.ts

```diff
@@ -38,3 +39,33 @@ function findFilesWalk(paths: string[], depth: number) {
 
   return [].concat(...foundPaths);
 }
+
+export const executableSuffix = platform.os === "win" ? ".exe" : "";
+
+// Returns true if the path exists.
+export function existsSync(path: string): boolean {
```

Comment:
existsSync is super useful. Maybe we should add it to the core API...
The comment should be jsdoc `/** Returns true if the path exists. */`